### DISCUSSION TITLE
CSV Importer only works for 25 lines maximum?

### DISCUSSION THREAD

**Discussion by TheOtherGuy** [ghostfolio/ghostfolio#2986] [Q&A]
I received an error when I tried to import a CSV file with >25 lines.
It will throw this error: _**Too many activities (25 at most)**_

Is this intended behaviour? Or is this a limitation of the free version?

**Comment by dtslvr** [comment]
Hi @TheOtherGuy

Yes, there is a limitation on the number of activities you can import within one request in the [Ghostfolio Basic](https://ghostfol.io/en/pricing) plan.

**Reply by AsheeHuang** [reply]
Could you put this information on official website ?
It is hard to find this limitation

**Reply by anshuvajpayee** [reply]
@dtslvr 
Please put this information  publicly.  Appreciate your efforts.
